src/test/resources/features/ForgotPasswordVS.feature

```diff
@@ -0,0 +1,24 @@
+@FP
+Feature: Automate test set "Forgot Password"
+  Background:
+    Given I navigate to the "login" page
+#    When I enter "ask_instr@aol.com" into email input
+#    And I enter"12345" into password
+#    When I click "Sign In"
+#    And I wait for 1 second
+
+@forgotPass
+  Scenario: Forgot Password
+    Then I click on "I forgot my password"
+    And I should see the header as "Reset Password Request"
+    When I click on "required" email field
```

Comment:
Please change the wording of the step:
When I enter "mavunaku@gmail.com" into email field